src/pages/surveys/createSurvery/index.tsx

```diff
@@ -0,0 +1,200 @@
+                <p className='font-medium text-3xl text-light-grey-200 pb-2'>
+                  {item.title}
+                </p>
+                {item.id === 'option format' ? (
+                  <Input
+                    placeHolder='Multiple Select'
+                    isCurved
+                    hasBorder
+                    className='!border-light-blue-50 !drop-shadow-none placeholder:text-[#1C1C1E]'
+                  />
+                ) : (
+                  <Input
+                    placeHolder='Enter title of the survey'
+                    isCurved
+                    hasBorder
+                    className='!border-light-blue-50 !drop-shadow-none placeholder:text-[#1C1C1E]'
+                  />
+                )}
+              </div>
+            </>
+          ))}
+          <div className='font-normal text-xl text-[#1C1C1E]'>
+            <div className='flex items-center gap-3 pt-3 '>
+              <input
+                type='radio'
+                className='w-8 h-8 border-[#8E8E93] border-2'
+              />
+              <span className=' w-full py-3 border-b border-b-[#C7C7CC] font-normal'>
+                Option 1
+              </span>
+            </div>
+            <div className='flex items-center gap-3 pt-3'>
+              <input
+                type='radio'
+                className='w-8 h-8 border-[#8E8E93] border-2'
+              />
+              <span className=' w-full py-3 border-b border-b-[#C7C7CC]'>
+                Option 2
+              </span>
+            </div>
+          </div>
+
+          <div className='flex items-start gap-3 font-semibold text-light-blue-main'>
+            <div className='flex gap-2 items-center justify-start'>
+              <CircledPlus />
+              <span>Add option</span>
+            </div>
+            <div className='flex gap-2 items-center justify-start'>
+              <SmallBin />
+              <span>Remove option</span>
+            </div>
+          </div>
+
+          <div
+            className=' absolute bottom-0 right-0
+          flex gap-2 items-center justify-start font-semibold text-light-blue-main'
+          >
+            <CircledPlus />
+            <span>Add option</span>
+          </div>
+        </div>
+      </Card>
+
+      <Card
+        hasHeader
+        hasBadge
+        hasBorder
+        className='!border-b-light-secondary-light_blue'
+        title='Others'
+      >
+        <>
+          <div className='grid grid-cols-[minmax(250px,480px)_minmax(250px,480px)] justify-between gap-5'>
+            {OtherList.map((item) => (
+              <>
+                <div className='max-w-[620px] mt-10 mb-2'>
+                  <p className='font-medium text-3xl text-light-grey-200 pb-2'>
+                    {item.title}
+                  </p>
+                  {item.id === 'points awarded (optional)' ? (
+                    <SearchSelect options={AwardOptions} />
```

Comment:
the placeholder color for your select is different from what's on the design @Maap014